tests/test_threads.py

```diff
@@ -180,3 +181,16 @@ def test_should_abort():
 
         should_abort()
         ThreadPool().query_abort.called_once_with()
+
+
+def test_stop_threads():
+    with patch("guikit.threads.ThreadPool", MagicMock()):
```

Comment:
If you patch the whole `ThreadPool` class, their methods will not behave as expected - they will do nothing. As a consequence, when you run the mocked method `run_thread`, no thread will really be created.